src/core/segment_reader.rs

```diff
@@ -69,17 +69,9 @@ impl SegmentReader {
         &self.schema
     }
 
-    /// Return the number of documents that have been
-    /// deleted in the segment.
-    pub fn num_deleted_docs(&self) -> DocId {
```

Comment:
Isn't this a public API someone may use currently?

Reply:
That's true. yes let's readd this.

Reply:
Ah we still need this method in demux